tequilapi/endpoints/service.go

```diff
@@ -164,11 +174,34 @@ func (se *ServiceEndpoint) ServiceStart(resp http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	se.serviceState = service.Running
+	optionsParser, ok := se.optionsParser[sr.ServiceType]
+	if !ok {
+		utils.SendErrorMessage(resp, "Invalid service type", http.StatusBadRequest)
+		return
+	}
+
+	options, err := optionsParser(sr.Options)
+	if err != nil {
+		utils.SendErrorMessage(resp, "Invalid options", http.StatusBadRequest)
+		return
+	}
+
+	if se.isAlreadyRunning(sr) {
+		utils.SendErrorMessage(resp, "Service already running", http.StatusConflict)
+		return
+	}
+
+	id, err := se.serviceManager.Start(identity.FromAddress(sr.ProviderID), sr.ServiceType, options)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusInternalServerError)
+		return
+	}
+
+	instance := se.serviceManager.Service(id)
 
 	resp.WriteHeader(http.StatusCreated)
-	statusResponse := toServiceInfoResponse(se.serviceState)
-	statusResponse.ID = "6ba7b810-9dad-11d1-80b4-00c04fd430c8"
+	statusResponse := toServiceInfoResponse(id, instance)
+	statusResponse.Status = string(service.Starting)
```

Comment:
`toServiceInfoResponse()` already has this fields mapping, dont map 2nd time

Reply:
Fixed.